dcp_prototype/backend/browser/README.md

```diff
@@ -0,0 +1,44 @@
+# Browser Backend
+
+This application serve as the API for the browser frontend.
+
+# Chalice App Template Instructions
+
+To deploy the app, type `make deploy` in this directory.
+
+Filename                  | Purpose                           | Information links
+--------------------------|-----------------------------------|------------------------------------------
+`./api/app.py`                  |The application entry point        | [Chalice Docs](https://chalice.readthedocs.io/en/latest/)
+`requirements-dev.txt`    |Developer environment dependencies | [Pip requirements files](https://pip.readthedocs.io/en/1.1/requirements.html)
+`./api/requirements.txt`        |Application dependencies           | [Chalice App Packaging](https://chalice.readthedocs.io/en/latest/topics/packaging.html)
+`Makefile`                |Tools for packaging and deploying  | [Automation and Make](https://swcarpentry.github.io/make-novice/)
+`./api/.chalice/config.in.json`	|A template for the Chalice config file for the app    | [Chalice Configuration File](https://chalice.readthedocs.io/en/latest/topics/configfile.html)
+`iam/policy-template/browser-lamnbda.json`|IAM policy for the app's IAM role  | [Lambda Permissions](https://docs.aws.amazon.com/lambda/latest/dg/intro-permission-model.html)
+`test/test.py`            |Test suite template                | [Python unittest](https://docs.python.org/3/library/unittest.html)
+
+## How to the chaluce app
```

Comment:
typo

Also, this section describes how to create a new app. I imagine instructions on how to deploy a new stage or manage existing stages would be more relevant